cfgen/cfgen

```diff
@@ -17,10 +17,48 @@ import sys
 from typing import (Any, Callable, Dict, Iterable, Iterator, List, NamedTuple,
                     Optional, Set, Tuple)
 import yaml
+import logging
 
 
 __version__ = '0.14'
 
+LOG = logging.getLogger('cfgen')
+
+
```

Comment:
Currently this decorator is used at one place only for run_command().
Reason for having a decorator is reusability.
Exception logic is not used inside run_command(), purposely.
The decorator can be moved to some common file like util/s later if needed for multiple such function calls outside this file.